### PR TITLE
Release v1.3.0

### PR DESCRIPTION
**Pull request by jdrew82**
This PR is to prep for the v1.3.0 release.

### CODE REVIEW

File: .cookiecutter.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "plugin_slug": "nautobot-ssot",
+    "project_slug": "nautobot-plugin-ssot",
+    "repo_url": "https://github.com/nautobot/nautobot-plugin-ssot",
+    "base_url": "ssot",
+    "min_nautobot_version": "1.4.0",
+    "max_nautobot_version": "1.9999",
+    "nautobot_version": "1.5.1",
+    "camel_name": "NautobotSsot",
+    "project_short_description": "Nautobot Single Source of Truth",
+    "version": "1.2.0",
```
Comment (glennmatthews) [inline .cookiecutter.json:18]: Should this be 1.3.0? I'm not entirely clear on the purpose of this file - do we need to keep this field in sync with the app version number?

File: .cookiecutter.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "cookiecutter": {
+    "codeowner_github_usernames": "@glennmatthews @jathanism @lampwins @chadell",
```
Comment (glennmatthews) [inline .cookiecutter.json:3]: Should this be `@nautobot/plugin-ssot` instead?

File: docs/admin/compatibility_matrix.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Compatibility Matrix
+
+While that last supported version will not be strictly enforced--via the max_version setting, any issues with an updated Nautobot supported version in a minor release, will require a bug to be raised and a fix in Nautobot core to address, with no fixes expected in this plugin. This allows the Nautobot Single Source of Truth App the ability to quickly take advantage of the latest features.
```
Comment (glennmatthews) [inline docs/admin/compatibility_matrix.md:3]: I think this is boilerplate text, but I honestly have no idea what this paragraph is telling me.

**Review by Kircheneer — APPROVED** [review]

**Review by glennmatthews — APPROVED** [review]

File: .cookiecutter.json (hunk 1 above)
Comment (jdrew82) [reply .cookiecutter.json:18]: This file is used to re-generate the cookie I believe. I'm not entirely sure it's required to be updated as we update the repo. Thoughts @Kircheneer and @chadell?

File: docs/admin/compatibility_matrix.md (hunk 3 above)
Comment (jdrew82) [reply docs/admin/compatibility_matrix.md:3]: This was already in the repo before I created this PR.

File: .cookiecutter.json (hunk 1 above)
Comment (Kircheneer) [reply .cookiecutter.json:18]: Paging @snaselj

File: .cookiecutter.json (hunk 2 above)
Comment (jdrew82) [reply .cookiecutter.json:3]: Again, this was generated when the cookie was first created and I'm not sure it needs to be updated as we update the project?

**Pull request merged**